### ISSUE TITLE
[Neon staging] Custodes misbehaving

### ISSUE THREAD

**Issue by JoyfulGen** [DDMAL/Neon2#1223]
I have observed two custos-related phenomena:

1. If you move a custos that is already on the page, it'll go somewhere else than where you want the _first_ time you move it; after that it behaves normally.

Demo:

https://github.com/DDMAL/Neon/assets/83373378/b7355550-645e-4745-afcd-f2a6ec6dd90d

2. If you input a new custos, it'll most probably assign to the wrong staff. In fact it looks like there are a couple of "dominant" staves that the custodes prefer, so a new custos will only assign to the closest dominant staff. You can see in the video that the first yellow staff is very popular until green has his moment and then orange takes over. Also, the further away the dominant staff is, the higher the chance that the custos won't go at all where you placed it. It also seems to be impossible to reassign the custos to the nearest staff.

Demo:
https://github.com/DDMAL/Neon/assets/83373378/0911cc9d-b99b-4d45-b2eb-a3f8e8a0c65e

**Comment by JoyfulGen** [comment]
UPDATE: The "dominant" staves issues doesn't apply only to custodes, so I've made a separate issue about it here: https://github.com/DDMAL/Neon/issues/1226

**Comment by yinanazhou** [comment]
This issue is fixed. Waiting on rism to merge changes to develop.
